vertical-pod-autoscaler/pkg/admission-controller/logic/pod_resource_handler_test.go

```diff
@@ -395,9 +396,19 @@ func TestGetPatchesForResourceRequest_TwoReplacementResources(t *testing.T) {
 				}`)
 	recommendAnnotations := vpa_api_util.ContainerToAnnotationsMap{}
 	frp := fakeRecommendationProvider{recommendResources, recommendAnnotations, "name", nil}
-	lc := limitrange.NewNoopLimitsCalculator()
-	s := NewAdmissionServer(&frp, &fppp, &fvpp, lc)
-	patches, err := s.getPatchesForPodResourceRequest(podJson, "default")
+	h := podResourceHandler{
```

Comment:
One more nit:  We can use `NewPodResourceHandler` to initialize structure. Here and everywhere else.

Reply:
Done.